graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -312,3 +378,4 @@ def enter_field_definition(self, node, *args):
             else:  # Make copy of node with the changed name, return the copy
                 field_node_with_new_name = get_copy_of_node_with_new_name(node, new_field_name)
                 return field_node_with_new_name
+        return None
```

Comment:
```suggestion

        return None
```

Reply:
If de-indenting two or more levels, I find it helps readability to add a blank line in between.